src/app/authentication/authentication-routing.module.ts

```diff
@@ -31,6 +32,10 @@ const routes: Routes = [
   {
     path :'dashboard/viewinvitation-details',
     component :ViewinvitationDetailsComponent
+  },
+  {
+    path :'dashboard/create-friendlist',
```

Comment:
create a different friends module similar to cards.
Have the route setup also similar to cards.
It path should be 'friends'.
Kindly refer to cards and replicate the same for friends.